CMakeLists.txt

```diff
@@ -229,7 +229,7 @@ ENDIF(WANT_TAP)
 
 # check for CARLA
 IF(WANT_CARLA)
-	PKG_CHECK_MODULES(CARLA carla-standalone>=1.9.5)
+	PKG_CHECK_MODULES(CARLA carla-native-plugin)
```

Comment:
As I mentioned in #4593, I think we can fall back to `carla-standalone` if `carla-native-plugin` is not available.
```CMake
IF(NOT CARLA_FOUND)
	PKG_CHECK_MODULES(CARLA carla-standalone>=1.9.5)
ENDIF()
```

Reply:
Done via b38dd02.